tested/dsl/translate_parser.py

```diff
@@ -720,7 +724,7 @@ def _convert_dsl_list(
     return objects
 
 
-def _convert_dsl(dsl_object: YamlObject) -> Suite:
+def convert_dsl(dsl_object: YamlObject) -> Suite:
```

Comment:
I don't think this is need, with the other suggestions I made

Reply:
I know. This change should not be needed if you follow the suggestion made in main

Reply:
This is related to the `to_yaml_object` change (and thus depends on the decision there)